core-api/src/main/java/com/optimizely/ab/config/ProjectConfigManager.java

```diff
@@ -23,5 +23,7 @@ public interface ProjectConfigManager {
      * @return ProjectConfig
      */
     ProjectConfig getConfig();
+
+    String getSDKKey();
```

Comment:
Can we get sdkKey via `getConfig().getSDKKey()` instead of adding the new inferface?
This will be convenient but we want to avoid any breaking changes with this new method here.

Reply:
see [this](https://github.com/optimizely/java-sdk/pull/501/files#r1067378831) comment. It will make sure that even when the config is not fetched, notificationCenter will contain the handler, which updates the ODPConfig.

Reply:
@mnoman09 Got it! Can we add a default implementation of this method (returning null), so no changes required for existing implementations. They'll all share the same internal notificationCenter for "null" sdkKey (until they fix it), which looks safe to me.
I believe the default interface method is supported in Java8 and also supported in android.